hadoop-ozone/common/pom.xml

```diff
@@ -82,7 +82,43 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
         <filtering>true</filtering>
       </resource>
     </resources>
+    <extensions>
```

Comment:
Is it required? I think it's already added to the parent pom and not required here...

Reply:
its no more present